app/lib/actions/katello/content_view_version/export.rb

```diff
@@ -0,0 +1,70 @@
+module Actions
+  module Katello
+    module ContentViewVersion
+      class Export < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        ISO_OUTPUT_BASEDIR = "/var/lib/pulp/published/yum/master/group_export_distributor/"
+
+        def plan(content_view_version, export_to_iso, since, iso_size)
+          # assemble data to feed to Pulp
+          start_date = since ? since.iso8601 : nil
+          content_view = ::Katello::ContentView.find(content_view_version.content_view_id)
+          org_label = ::Organization.find_by(:id => content_view.organization_id).label
+          repo_pulp_ids = content_view.repositories.collect { |r| r.pulp_id }
```

Comment:
You're still grabbing the repositories from the content view, but i think you want the repos from the version as well as the content view puppet environment.  something like:
```
repo_pulp_ids = content_view_version.first.archived_repos.pluck(&:pulp_id)
repo_pulp_ids << content_view_version.first.archive_puppet_environment.pulp_id if content_view_version.first.archive_puppet_environment
```